admin/src/composables/useDateLocale.js

```diff
@@ -0,0 +1,10 @@
+import { useI18n } from 'vue-i18n'
+import { de, enUS as en, fr, es, nl } from 'date-fns/locale'
+
+const locales = { en, de, es, fr, nl }
+
+export function useDateLocale() {
+  const { locale } = useI18n()
+  const dateLocale = locales[locale.value] || en
```

Comment:
if attribute 'locales' is a hard coded array of locale-values and for the object 'locale' it is not sure to have a correct initialized value, which will match one of the 'locales' entries, then an index-out-of-bount exception will occur, right?
So i think a check if 'locales' contains an entry with 'locale.value' will be more stabil...

Reply:
Do you know how you usually check in js if an object contain an attribute?
```js
let dateLocale
if (locales[locale.value] !== undefined) {
  dateLocale = locales[locale.value]
  } else {
  dateLocale = en
}
```
Shorter is:
```js
const dateLocale = locales[locale.value] || en
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR